### ISSUE TITLE
Rate limit warning on renew & replace cert is CA-specific and might not apply to all CAs

### ISSUE THREAD

**Issue by dmmortimer** [certbot/certbot#3456]
When you run certbot and a matching certificate already exists, you see this message:

```
You have an existing certificate that contains exactly the same domains you requested and isn't close to expiry.
(ref: /etc/letsencrypt/renewal/mortimer.com.conf)

What would you like to do?
-------------------------------------------------------------------------------
1: Keep the existing certificate for now
2: Renew & replace the cert (limit ~5 per 7 days)
-------------------------------------------------------------------------------
```

Problem, the rate limit is CA-specific and might not apply when running against a different CA.

I'd like to suggest a text change from
limit ~5 per 7 days
To
may be subject to CA rate limits

If this is likely to be accepted let me know and I'll submit a pull request.

**Comment by bmw** [comment]
Making this change would certainly be controversial among the Certbot team. On the one hand, Certbot is not Let's Encrypt specific, so we shouldn't be hard coding information/assumptions based on their services. On the other hand, Let's Encrypt remains the only functioning ACME CA and things like rate limits are very useful information for our users.

I'm personally inclined to leave it alone until another CA pops up.

**Comment by dmmortimer** [comment]
Ok, maybe that won't be too long. But it seems to me that even in a world with only one ACME CA, it would be nice to be able to change CA policy such as rate limits without causing client UI to become out of date. I suppose the tilde is there to indicate that the actual values may change.

**Comment by bmw** [comment]
I'd personally be willing to accept a PR making this change.

**Comment by mnordhoff** [comment]
This is still a thing, stale bot.

**Comment by dmmortimer** [comment]
i'll submit a pr for this ui text change - how do i get permission to create a branch?

**Comment by alexzorin** [comment]
Closing as it was fixed by #8061.
